src/App.tsx

```diff
@@ -1,22 +1,8 @@
-import * as React from "react"
-import {
-  ChakraProvider,
-  Box,
-  Text,
-  Link,
-  VStack,
-  Code,
-  Grid,
-  theme,
-} from "@chakra-ui/react"
-import { ColorModeSwitcher } from "./ColorModeSwitcher"
-import { Logo } from "./Logo"
+import { ChakraProvider, theme } from "@chakra-ui/react";
+import Sidebar from "./components/Sidebar";
 
 export const App = () => (
   <ChakraProvider theme={theme}>
-    <Box textAlign="center" fontSize="xl">
-      <Text>Time Tracker</Text>
-      <Text>Built with ChakraUI</Text>
-    </Box>
+    <Sidebar />
```

Comment:
It might make more sense to rename this to something like `MainScreen` since it not only contains the sidebar, but also navbar with the body for the selected goal

Reply:
I agree we could refactor that